src/server/docker/gunicorn_conf.py

```diff
@@ -0,0 +1,67 @@
+import json
+import multiprocessing
+import os
+
+workers_per_core_str = os.getenv("WORKERS_PER_CORE", "1")
+max_workers_str = os.getenv("MAX_WORKERS")
```

Comment:
I am not very familiar with `gunicorn` configuration, was just following patterns from [here](https://github.com/vincentsarago/containers/tree/main/uvicorn-gunicorn) but I created an image from scratch since those images are ~800 MB.

I'll do some more digging and add some documentation around the configuration here.